### ISSUE TITLE
How can we make Gutenberg better for those with color blindness?

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#2825]
This is just something that came up when testing Gutenberg. I used an app to see what people with various forms of color blindness see. The one that really was shocking was monochromacy. If you look at this, it's what you'd see:

![2017-09-28 at 15 30](https://user-images.githubusercontent.com/253067/30972631-090d289e-a463-11e7-8e99-117d307769a8.png)

There is little to no indication beyond text on buttons, there is also little to not indication of colors on the palette. Even hovering doesn't show what colors. Just adding a hover there could to me help this issue at least in part.

**Comment by hedgefield** [comment]
You're talking about the fact that buttons without an outline or underline don't register as buttons in monochrome, and that the colors don't have a hover hint? The second seems easy to fix, the first issue would need some exploration.

Instead of just changing the color of the letters on hover, showing a button outline on hover would seem like a quick step in the right direction. It also might be worth exploring giving all those shapeless buttons a shape to begin with, even if it's just something like the Block Alignment buttons have in this example for instance.

**Comment by laras126** [comment]
I think @karmatosed is talking about a text representation of what the palette colors are, either as a tooltip or selection. There's a good outline of this issue in [a recent Smashing Magazine article](https://www.smashingmagazine.com/2017/10/nailing-accessibility-design/#color-as-a-tool-but-not-a-magic-key) (scroll a little to the Nike/Amazon color UI discussion).  
Perhaps something as simple as this would help?
![Color label](https://i.imgur.com/GQdj4oT.png)
I'd also suggest having an outline _around_ the selected color dot rather than making it look empty as it currently is.

**Comment by afercia** [comment]
> There is little to no indication beyond text on buttons, there is also little to not indication of colors on the palette. 

This is one of the reasons why a visible label (plain text) is always the best option. Re: the color palette, colors would need a name though, see https://github.com/WordPress/gutenberg/issues/2699
> ideally, colors should have a human-understandable named color.

**Comment by karmatosed** [comment]
For now lets close as we can deal with tooltips.
